app/presenters/menu/item.rb

```diff
@@ -10,7 +10,7 @@ def name
     end
 
     def visible?
-      ApplicationHelper.role_allows_intern(rbac_feature)
+      ApplicationHelper.role_allows(rbac_feature)
```

Comment:
Nope, that's the method in ApplicationHelper and what all the front end callers use.